packages/breadcrumbs/src/styled/StyledBreadcrumb.ts

```diff
@@ -13,12 +13,12 @@ const COMPONENT_ID = 'breadcrumbs.breadcrumb_view';
 /**
  * Accepts all `<nav>` props
  */
-const BreadcrumbView = styled.nav.attrs({
+const StyledBreadcrumb = styled.nav.attrs({
```

Comment:
For all styled-components we are using named exports. Change this to `export const StyledBreadcrumb`